src/blocks/icons/components/Refresh.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import { IconWrapper } from '../IconWrapper';
+import { IconProps } from '../Icons.types';
+
+const Refresh: FC<IconProps> = (allProps) => {
+  const { svgProps: props, ...restProps } = allProps;
+  return (
+    <IconWrapper
+      componentName="Refresh"
+      icon={
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="inherit"
+          height="inherit"
+          viewBox="0 0 26 26"
+          fill="none"
```

Comment:
```suggestion
          fill="none"
          {,,,props}
```

Btw This component already exists